README.md

```diff
@@ -23,17 +24,20 @@ docs*
 
 Name | Overhead (release) | Build (debug) | Parse (release) | Invalid UTF-8 | Downloads | Version
 -----|--------------------|---------------|-----------------|---------------|-----------|--------
-null | 0 KiB | 695ms | 2ms | Y | ![Download count](https://img.shields.io/crates/dr/None) | -
-argh | 30 KiB | 6s | 2ms | N | ![Download count](https://img.shields.io/crates/dr/argh) | v0.1.7
-clap | 678 KiB | 6s | 3ms | Y | ![Download count](https://img.shields.io/crates/dr/clap) | v3.0.0
-clap-minimal | 633 KiB | 6s | 2ms | Y | ![Download count](https://img.shields.io/crates/dr/clap) | v3.0.0
-clap_derive | 687 KiB | 14s | 2ms | Y | ![Download count](https://img.shields.io/crates/dr/clap) | v3.0.0
-gumdrop | 24 KiB | 6s | 3ms | N | ![Download count](https://img.shields.io/crates/dr/gumdrop) | v0.8.0
-lexopt | 27 KiB | 934ms | 2ms | Y | ![Download count](https://img.shields.io/crates/dr/lexopt) | v0.2.0
-pico-args | 23 KiB | 941ms | 3ms | Y | ![Download count](https://img.shields.io/crates/dr/pico-args) | v0.4.2
-xflags | 23 KiB | 2s | 3ms | Y | ![Download count](https://img.shields.io/crates/dr/xflags) | v0.2.3
-
-*System: Linux 5.4.0-84-generic (x86_64) w/ `-j 8`*
+null | 0 KiB | 818ms | 2ms | Y | ![Download count](https://img.shields.io/crates/dr/None) | -
```

Comment:
Could you revert the benchmark numbers.  I think I'd prefer to know all the conditions they were run under